doc/source/contributing_code.rst

```diff
@@ -202,8 +199,14 @@ This sets the number of files to 4096. However, this command only applies to you
 Building MDAnalysis
 -------------------
 
-Make sure that you have :ref:`cloned the repository <forking-code-repo>`  
-and activated your virtual environment. First we need to install dependencies. If you're using conda, you'll need a mix of conda and pip installations:
+With conda
+----------
+
+.. note::
+    Make sure that you have :ref:`cloned the repository <forking-code-repo>`  
+    and activated your virtual environment with :code:`conda activate mdanalysis-dev`.
+    
+First we need to install dependencies. You'll need a mix of conda and pip installations:
 
     .. code-block:: bash
 
```

Comment:
I think we can make the switch from biobuilds to bioconda as long as we're happy that the latter is suitably secure/well-used? (which I think it is)